package.json

```diff
@@ -4,9 +4,9 @@
 	"description": "A Chrome extension for the Khan Academy Computer Programming section",
 	"private": true,
 	"scripts": {
-		"build": "esbuild ./src/index.ts ./src/background.ts ./src/content.ts ./src/popup.ts --bundle --outdir=dist --target=es2018",
```

Comment:
This is fine because our minimum Chrome version is 88.

Reply:
Yeah, [`BigInt`](https://caniuse.com/bigint) is supported on all Chrome versions > 66. Moreover, Manifest V3 is only supported on versions >= 88.